frontends/precinct-scanner/src/screens/unlock_admin_screen.test.tsx

```diff
@@ -45,21 +40,15 @@ test('authentication', async () => {
   screen.getByText('• • • • • •');
 
   await waitFor(() =>
-    expect(attemptToAuthenticateUser).toHaveBeenNthCalledWith(1, '012345')
+    expect(auth.checkPasscode).toHaveBeenNthCalledWith(1, '012345')
   );
+  screen.getByText('- - - - - -');
+});
 
+test('If passcode is incorrect, error message is shown', () => {
+  const auth = Inserted.fakeCheckingPasscodeAuth({
+    wrongPasscodeEntered: true,
+  });
+  render(<UnlockAdminScreen auth={auth} />);
```

Comment:
Any reason why you chose to test this case by mocking an auth already in the wrong-passcode state as opposed to actually entering an invalid code?